services/frontend/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "gatsby-starter-hello-world",
+  "private": true,
+  "description": "A simplified bare-bones starter for Gatsby",
+  "version": "0.1.0",
+  "license": "0BSD",
+  "scripts": {
+    "build": "gatsby build",
+    "develop": "gatsby develop",
+    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
+    "start": "npm run develop",
+    "serve": "gatsby serve",
+    "clean": "gatsby clean",
+    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
+  },
+  "dependencies": {
+    "gatsby": "^2.24.54",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0"
+  },
+  "devDependencies": {
+    "gatsby": "^2.24.60",
+    "prettier": "2.1.1"
```

Comment:
Not needed here since it's installed globally

Reply:
I do think there are specific ESLint configurations for React though that we might wanna set up. I can help configure that if you want

Reply:
Sure, I haven't been getting issues, but a bit more detailed style guide wouldn't hurt